src/ccstruct/tabletransfer.h

```diff
@@ -0,0 +1,60 @@
+/******************************************************************************
+ * File:        tabletransfer.h
+ * Description: Infrastructure for the transfer of table detection results
+ * Author:      Sintun
+ *
+ * (C) Copyright 2021, Sintun
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * http://www.apache.org/licenses/LICENSE-2.0
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ *
+ ****************************************************************************/
+
+#pragma once
+#include <memory>
+#include <vector>
+#include "rect.h"
+
+
+/// Structure for data transfer from table detector 
+struct MyTable {
```

Comment:
This is a very generic name. Maybe a more Tesseract specific name would be better, for example `TessTable`.

Reply:
All other header files use `namespace tesseract`. Would it be reasonable for the new header files here, too?

Reply:
The global namespace should be avoided, that is correct.
I Added the tesseract namespace.